### ISSUE TITLE
Anti-H2A-H2B antibody positivity

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6614]
**Preferred term label:** Anti-H2A-H2B antibody positivity

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
Anti-histone positivity

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
please see comment for https://github.com/obophenotype/human-phenotype-ontology/issues/6615

**Comment by eswietlik** [comment]
@pnrobinson as with anti-H3-H4 Ab this antibody is specific against the dimer.

**Comment by pnrobinson** [comment]
added term
